code/game/machinery/PDApainter.dm

```diff
@@ -86,12 +86,10 @@
 			user.visible_message("[user] is repairing [src].", \
 							"<span class='notice'>You begin repairing [src]...</span>", \
 							"<span class='italics'>You hear welding.</span>")
-			playsound(loc, O.usesound, 40, 1)
-			if(O.use_tool(src, user, 40))
+			if(O.use_tool(src, user, 40, volume=50))

```

Comment:
Aren't these missing the amount arg?

Reply:
Basic tools, such as wire cutter or multitool, don't support non-zero `amount` as there is no meaning in `amount` for them. It only works on tools or items that have amount or charges (welders, stacks).